docs/source/running-jobs/environment-variables.rst

```diff
@@ -108,6 +108,9 @@ Environment variables for ``setup``
   * - ``SKYPILOT_SETUP_NODE_IPS``
     - A string of IP addresses of the nodes in the cluster with the same order as the node ranks, where each line contains one IP address.
     - 1.2.3.4
+  * - ``SKYPILOT_SERVE_REPLICA_ID``
```

Comment:
```suggestion
  * - ``SKY_SERVE_REPLICA_ID``
```
to discuss; not sure

Reply:
Let's keep `SKYPILOT` to be align with the other environment variables.